aula-pratica-04/ex5.c

```diff
@@ -0,0 +1,28 @@
+/*
+    Faça uma rotina que recebe um string qualquer e retorna-o invertido.
+    Por exemplo, se a string
+    passada por parâmetro fosse "teste", a rotina deve retornar "etset".
+*/
+
+#include<stdio.h>
+#include<string.h>
+
+void inverter_string(char str[100]) {
+    int size = strlen(str) - 1;
+    char str_reversa[100];
+    for(int i = size; i >= 0; i--) {
+        str_reversa[size - i] = str[i];
+    }
+    printf("String invertida: %s\n", str_reversa);
```

Comment:
(opcional) é sempre legal fazer funções reaproveitaveis neh? o que vc acha de transformar esse print em um `return`?
```suggestion
    return str_reversa;
```

Reply:
Pra retornar uma _string_ assim na função, eu não preciso usar um ponteiro?
Pelo o que andei pesquisando, precisaria fazer algo tipo isso:
```c
const char * myFunction()
{
    return "My String";
}
```

Reply:
é, acho que ficaria assim mesmo :D, que que cê acha? acha que faz sentido, acha que fica melhor?